pkg/github/github.go

```diff
@@ -104,7 +104,7 @@ func (g *GitHub) getTags(ctx context.Context, owner, name, filter string) ([]str
 	const query = `
 		query($owner:String!, $name:String!, $filter:String!, $after:String) {
 		  repository(name: $name, owner: $owner) {
-		    refs(first: 100, refPrefix: "refs/tags/", after: $after, query: $filter, direction: ASC) {
+		    refs(first: 100, refPrefix: "refs/tags/", after: $after, query: $filter, direction: DESC) {
```

Comment:
btw if you use `first: 1` - you can drop the function which returns the latest, right?)

Reply:
My thinking was you can release `1.2.1` after `1.3.0`. TBH that shouldn't happen with config, but still I feel better that way.